modules/text2vec-cohere/clients/cohere_test.go

```diff
@@ -20,14 +20,57 @@ import (
 	"testing"
 	"time"
 
+	"github.com/weaviate/weaviate/entities/modulecapabilities"
+
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"github.com/sirupsen/logrus/hooks/test"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/weaviate/weaviate/modules/text2vec-cohere/ent"
 )
 
+type FakeClassConfig struct {
```

Comment:
please make this class a private one `fakeClassConfig` and place it at the end of the file so that the first thing that a user will see in this file is a test